tests/api/Jalasoft.TeamUp.Projects.API.Tests/GetProjectByIdTests.cs

```diff
@@ -46,13 +46,13 @@ public void GetProject_Returns_NotFoundResult()
         {
             // Arrange
             var request = this.mockHttpContext.Request;
-            this.mockService.Setup(service => service.GetProject(Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"))).Throws(new ProjectsException(ProjectsException.ProjectsErros.NotFound));
+            this.mockService.Setup(service => service.GetProject(Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"))).Throws(new ProjectsException(ProjectsException.ProjectsErrors.NotFound));
 
             // Act
             var response = this.getProject.Run(request, new Guid("5a7939fd-59de-44bd-a092-f5d8434584de"));
 
             // Assert
-            var notfountObjectResult = Assert.IsType<ContentResult>(response);
+            var notfountObjectResult = Assert.IsType<ObjectResult>(response);
         }
     }
```

Comment:
please add unit tests for the bad request response in the API and in the service

Reply:
done